src/metabase/driver/sql_jdbc.clj

```diff
@@ -174,3 +176,56 @@
   (let [sql (driver.sql/set-role-statement driver role)]
     (with-open [stmt (.createStatement ^Connection conn)]
       (.execute stmt sql))))
+
+(defmethod driver/upload-type->parser [:sql-jdbc :metabase.upload/varchar-255]
```

Comment:
I find it's easier to read if you use ns aliases
```suggestion
(defmethod driver/upload-type->parser [:sql-jdbc ::upload/varchar-255]
```

Reply:
fixed in mysql.clj, but upload isn't `:require`d in sql_jdbc.clj and I don't think I should add the dependency to save on the typing.